docs/wiki/[Variables]-deploy_management_resources.md

```diff
@@ -24,24 +24,65 @@ A full list of default Management Groups:
 ```hcl
     data "azurerm_client_config" current {}
 
-    module "enterprise_scale" {
```

Comment:
Please take a look at the existing variable pages as I don't believe this usage example is consistent with what we've been using. Will discuss offline.